src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import java.util.List;
+import java.util.Map;
+
+public interface FruitShopService {
```

Comment:
rename it to FruitService as it is working with fruits not with fruitShop

Reply:
done